games-action/prismlauncher/prismlauncher-9999.ebuild

```diff
@@ -122,12 +118,6 @@ src_prepare() {
 		elog "and select it with \"eselect java\", then recompile ${PN}."
 		eapply "${FILESDIR}/${PN}-8.2-openjdk21.patch"
 	fi
-
-	sed -i -e 's/-Werror//' CMakeLists.txt || die 'Failed to remove -Werror via sed'
-
-	# Prevent conflicting with the user's flags
-	# See https://bugs.gentoo.org/848765 and https://bugs.gentoo.org/911858 for more info
-	sed -i -e "/CMAKE_CXX_FLAGS_RELEASE/d" CMakeLists.txt || die 'Failed to remove "CMAKE_CXX_FLAGS_RELEASE" from CMakeLists via sed'
```

Comment:
I think we still need this: https://github.com/PrismLauncher/PrismLauncher/blob/0041d2a126ae6f35ce1ca2bb5c793ec5853c4343/CMakeLists.txt#L93

Reply:
If you use Gentoos own build type of RelWithDebInfo it means it wont use CMAKE_CXX_FLAGS_RELEASE or CMAKE_CXX_FLAGS_DEBUG. So removing this workaround comes free with removing the debug use flag.

Reply:
That makes sense, thanks

Reply:
Alright, Ill re add them with comment explaining this.